apps-rendering/src/components/Layout/StandardLayout.tsx

```diff
@@ -134,6 +135,16 @@ const StandardLayout: FC<Props> = ({ item, children }) => {
 						<Metadata item={item} />
 						<Logo item={item} />
 					</section>
+
+					{item.design === ArticleDesign.Explainer &&
+						item.outline.length > 0 && (
+							<section css={articleWidthStyles}>
+								<TableOfContents
```

Comment:
should this also be added to the Analysis layout?

Reply:
ignore this, I remembered the PR description specifying Explainer articles only

Reply:
This is a good question, I was also not sure and because I couldn't find any Analysis article with h2 tags, I assumed they may not need the TOC. I'll talk to Harry tomorrow to confirm, thanks for pointing out 👍

Reply:
Harry in on holiday, and we aren't sure if he wants the table of content on analysis articles at this stage or not. So to be on the safe side, we can just release the explainer version until he's back